waku/v2/node/wakunode2.nim

```diff
@@ -14,6 +14,7 @@ import
   ../protocol/waku_store/waku_store,
   ../protocol/waku_swap/waku_swap,
   ../protocol/waku_filter/waku_filter,
+  ../utils/peers,
```

Comment:
a minor comment, do we follow any naming conventions to distinguish utils modules from others? e.g., peers_utils or something like this

Reply:
As far as I can tell, the (current) convention is simply to group these under a `utils` subfolder. I think there's scope to improve how we handle utils in general, so watch this space. :)